src/test/java/com/wikia/webdriver/common/logging/PageObjectLogging.java

```diff
@@ -334,5 +416,8 @@ private void appendShowHideButtons() {
   @Override
   public void onFinish(ITestContext context) {
     CommonUtils.appendTextToFile(logPath, "</body></html>");
+
+    Gson gson = new Gson();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "gson" which hides the field declared at line 63. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AHiddenFieldCheck)